login.php

```diff
@@ -92,13 +93,13 @@ function cleanUp($string) {
                 $_SESSION['sTab']       = 't_sum';
                 $_SESSION['id']         = $id;
                 
-	        header ("Location: index.php?id=$id");
+	        header ("Location: $styleSelect?id=$id");
             } else {
-                $err = 'The user name or password is incorrect.';
+                $err = '<br>Wrong - Try again<br><br>';
                 $focus = 'username';
             }
         } else {   
-            $err = 'The user name or password is incorrect.';
+            $err = '<br>Wrong - Try again<br><br>';
```

Comment:
My preference.  I didn't like the wording or location of the submit button and error message.